posthog/api/capture.py

```diff
@@ -100,15 +100,11 @@ def get_event(request):
     timer.start()
     now = timezone.now()
     try:
-        data_from_request = load_data_from_request(request)
-        data = data_from_request["data"]
-    except TypeError:
+        data = load_data_from_request(request)
```

Comment:
Did not review any changes to `capture.py` or `test_capture.py` just yet, as I believe both of these modules are fully deprecated, can you please confirm? In which case it might be worth adding a matching PR in the plugin server. CC @Twixes @mariusandra

Reply:
Nope, nope, nope, they are definitely not deprecated – `process_event.py` parts will be purged, but the capture endpoint is till very much Django.